app/jobs/hyrax/background_job.rb

```diff
@@ -0,0 +1,7 @@
+module Hyrax
+  # A common base class for all Hyrax jobs.
+  # This allows upstream applications to manipulate all the hyrax jobs by
```

Comment:
Or if stream-directionality is confusing, "Hyrax consumers" is what we mean here.